### ISSUE TITLE
NameError: name 'StanzaLanguage' is not defined

### ISSUE THREAD

**Issue by connerxyz** [microsoft/presidio#750]
**Describe the bug**

Trying to use Stanza NLP engine with simple example and it throws an error.

```
import stanza
stanza.download("en")

from presidio_analyzer.nlp_engine import StanzaNlpEngine
StanzaNlpEngine(models={"en": "en"})
```

Above throws...

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../presidio_analyzer/nlp_engine/stanza_nlp_engine.py", line 41, in __init__
    for lang_code, model_name in models.items()
  File ".../presidio_analyzer/nlp_engine/stanza_nlp_engine.py", line 41, in <dictcomp>
    for lang_code, model_name in models.items()
NameError: name 'StanzaLanguage' is not defined
```

**To Reproduce**

**Expected behavior**

I would expect the `StanzaNlpEngine` to be able to be instantiated using the `models` argument provided.

**Screenshots**

**Additional context**

**Comment by omri374** [comment]
Hi @connerxyz, what version of `presidio-analyzer` are you using?

**Comment by connerxyz** [comment]
We're running 2.2.1. What version did your bug fix end up in? Thanks.

**Comment by omri374** [comment]
It was fixed in 2.2.2 https://github.com/microsoft/presidio/releases/tag/2.2.2. Closing for now
